### PR TITLE
Notification list

### PR DESCRIPTION
**Pull request by yoshiki1009**
create a basic notification function
when some user posts product that belongs to your group, a notification list is displayed on the notification screen.
You can go to the notification list from the top right bell icon.

issue: I am struggling with how to hide notification tab from the bottom tab bar, so I put it there temporarily. 
          When the user jumps to the product page from a notification list, sometimes it works, but sometimes it failed...

next:  The feature that once the user checked the notification, that list is removed from the lists is coming later

### CODE REVIEW

File: client/src/components/Header.js [hunk 1]
```diff
@@ -13,7 +15,9 @@ export default () => {
           <Image source={require('../assets/icons/logo-lg.png')} alt="logo" h="48px" w="112px" resizeMode="contain" />
 
           <Center size="8">
-            <NotificationIcon width="20px" />
+            <Pressable onPress={() => navigation.navigate('NotificationTab', { screen: 'NotificationStacks' })}>
```
Comment (satoshinbp) [inline client/src/components/Header.js:18]:
```suggestion
            <Pressable onPress={() => navigation.navigate('Notification')}>
```

File: client/src/navigators/NotificationStacks.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+const NotificationStack = createNativeStackNavigator()
+
+export default () => {
+  return (
+    <NotificationStack.Navigator screenOptions={{ headerShown: false }}>
+      <NotificationStack.Screen name="Notification" component={Notification} />
+    </NotificationStack.Navigator>
+  )
+}
```
Comment (satoshinbp) [inline client/src/navigators/NotificationStacks.js:13]: Delete this file

File: client/src/navigators/Tabs.js [hunk 3]
```diff
@@ -7,6 +7,7 @@ import GroupsStacks from './GroupsStacks'
 import ScanStacks from './ScanStacks'
 import FavsStacks from './FavsStacks'
 import ProfileStacks from './ProfileStacks'
+import NotificationStacks from './NotificationStacks'
```
Comment (satoshinbp) [inline client/src/navigators/Tabs.js:10]:
```suggestion
```

File: client/src/navigators/Tabs.js [hunk 4]
```diff
@@ -52,6 +53,7 @@ export default () => {
         component={ProfileStacks}
         options={{ tabBarLabel: 'Profile', tabBarIcon: ({ width }) => <ProfileIcon width={width} /> }}
       />
+      <Tab.Screen name="NotificationTab" component={NotificationStacks} />
```
Comment (satoshinbp) [inline client/src/navigators/Tabs.js:56]: import Notification screen

```suggestion
      <Tab.Screen name="Notification" component={Notification} />
```

File: client/src/screens/Notification.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+import { FlatList, Avatar } from 'native-base'
+
+import ListItemBarColored from '../elements/ListItemBarColored'
+
+export default () => {
+  const { user } = useSelector((state) => state.auth)
+  const notifications = user.notifications
+  const navigation = useNavigation()
+
+  const onPress = (item) => navigation.navigate('MyProductsTab', { screen: 'MyProduct', params: { id: item._id } })
```
Comment (satoshinbp) [inline client/src/screens/Notification.js:13]: item._id is notidfication id, not product id. Need to store product id in notification object and pass it to navigate params

**Review by satoshinbp — CHANGES_REQUESTED** [review]
I understand that there will be only one screen for notification. In that case, we don't need stack navigator.

**Comment by yoshiki1009** [comment]
fix and refactored notification function

**Review by satoshinbp — APPROVED** [review]

**Pull request merged**